brands/amenity/bank.json

```diff
@@ -4470,6 +4518,17 @@
       "name": "Security Bank"
     }
   },
+  "amenity/bank|Servus Credit Union": {
+    "countryCodes": ["ca"],
+    "matchNames": ["servus"],
```

Comment:
Is it a good fit for `short_name`?

Reply:
This can be an `alt_name`; it appears this way on some branches’ signs.